server/Dockerfile

```diff
@@ -3,13 +3,7 @@ FROM ubuntu:trusty
 
 # install python3 and build dependencies for c modules
 # update pip and distribute
-RUN apt-get update && \
-DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-python3 python3-dev python3-pip \
-build-essential libffi-dev git mercurial \
-libtiff5-dev libjpeg8-dev zlib1g-dev \
-libfreetype6-dev liblcms2-dev libwebp-dev \
-curl
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends python3 python3-dev python3-pip build-essential libffi-dev git mercurial libtiff5-dev libjpeg8-dev zlib1g-dev libfreetype6-dev liblcms2-dev libwebp-dev curl
```

Comment:
I agree but the deleted one was bad interpreted. The ` \` was removed without spaces between words.